### PR TITLE
Dropdown component

### LINKED ISSUE #158: Make dropdown component

### LINKED ISSUE #228: Move logout button to 'Your Name' dropdown in the navbar.

### PR DESCRIPTION
**Pull request by sambehrens**
closes #158 
closes #228 
- [x] Make optional arrow in dropdown

### CODE REVIEW

File: client/src/components/layout/Navbar.js [hunk 1]
```diff
@@ -21,6 +23,35 @@ export class Navbar extends Component {
                     <li className="">
                         <Link to="/dashboard">Poodl</Link>
                     </li>
+                    <li className="right" style={{ marginTop: '12px' }}>
```
Comment (karlmolina) [inline client/src/components/layout/Navbar.js:26]: why are you putting style inside the component

**Review by karlmolina — CHANGES_REQUESTED** [review]

**Review by karlmolina — CHANGES_REQUESTED** [review]
Does not show hand click cursor when you hover over username dropdown or any button in the dropdown.

File: client/src/assets/stylesheets/ui/dropdown.scss [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    }
+
+    hr {
+      border-color: $gray-3;
+      margin: 5px 0;
+      border-top: 0;
+      border-style: solid;
+    }
+
+    .dropdown-content-row {
```
Comment (karlmolina) [inline client/src/assets/stylesheets/ui/dropdown.scss:83]: add background-color: transparent here and the background is not gray anymore

**Review by karlmolina — CHANGES_REQUESTED** [review]

File: client/src/assets/stylesheets/ui/dropdown.scss (hunk 2 above)
Comment (sambehrens) [reply client/src/assets/stylesheets/ui/dropdown.scss:83]: ok I did. check and see if it looks good now.

**Comment by sambehrens** [comment]
@karlmolina I made the pointer happen on the dropdown rows, but I wasn't sure if we wanted it on the button, because you typically only want that pointer if it is a link, and the button is not a link, it just toggles the dropdown

**Review by karlmolina — APPROVED** [review]

**Pull request merged**